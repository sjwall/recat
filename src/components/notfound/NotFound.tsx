import React from 'react';
import { Translation } from 'react-i18next';
import Box from '@mui/material/Box';
import Paper from '@mui/material/Paper';
import Typography from '@mui/material/Typography';

import styles from './NotFound.module.css';

/**
 * 404 Not Found page
 *
 * @returns React component.
 */
export function NotFound() {
  return <Box className={styles.container}>
    <Paper elevation={3} className={styles.contents}>
        <Typography
          variant="h6"
          component="div"
          className={styles.item}
        >
          <Translation>
            {(t) => t('not_found')}
          </Translation>
        </Typography>
    </Paper>
  </Box>
}
