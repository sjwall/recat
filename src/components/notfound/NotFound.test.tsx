import React from 'react';
import renderer from 'react-test-renderer';
import { NotFound } from './NotFound';

describe('NotFound', () => {
  test('renders default', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
