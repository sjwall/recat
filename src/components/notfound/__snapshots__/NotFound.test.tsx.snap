// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders default 1`] = `
<div
  className="container MuiBox-root css-0"
>
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation3 contents css-3su884-MuiPaper-root"
  >
    <div
      className="MuiTypography-root MuiTypography-h6 item css-2ulfj5-MuiTypography-root"
    >
      The requested resource was not found.
    </div>
  </div>
</div>
`;
